sapi/fpm/fpm/fpm_scoreboard.c

```diff
@@ -210,6 +210,64 @@ void fpm_scoreboard_release(struct fpm_scoreboard_s *scoreboard) {
 	scoreboard->lock = 0;
 }
 
+struct fpm_scoreboard_s *fpm_scoreboard_copy(struct fpm_scoreboard_s *scoreboard, int copy_procs)
+{
+	struct fpm_scoreboard_s *scoreboard_copy;
+	struct fpm_scoreboard_proc_s *scoreboard_proc_p;
+	size_t scoreboard_size, scoreboard_nprocs_size;
+	int i;
+	void *mem;
+
+	if (!scoreboard){
+		scoreboard = fpm_scoreboard_get();
+	}
+
+	if (copy_procs){
+		scoreboard_size        = sizeof(struct fpm_scoreboard_s) + (scoreboard->nprocs) * sizeof(struct fpm_scoreboard_proc_s *);
```

Comment:
NIT: if you swap it with the next line, it could be:
```
scoreboard_size = sizeof(struct fpm_scoreboard_s) + scoreboard_nprocs_size
```
Also no point to align `=` (multiple spaces before that)

Reply:
Are you sure about this?

My understanding is that I need memory for three things:
1. fpm_scoreboard_s
2. an array of pointers to `fpm_scoreboard_proc_s`
3. an array of fpm_scorescoreboard_proc_s

Memory for 1. and 2. is included in `scoreboard_size`.
Memory for 3. is included in `scoreboard_nprocs_size`

What you're suggesting is not to allocate memory for 2.

By the way: The code for `fpm_scoreboard_copy` is based on `fpm_scoreboard_init_main`.

Reply:
Ah sorry you are right. It would be probably better to not have an array of pointers and embed it to the struct instead but that's not in the scope for this PR and should not go to the stable version as it's more improvement I guess.

Anyway ignore my comment, your version is fine.